jboss/container/wildfly/launch/access-log-valve/module.yaml

```diff
@@ -0,0 +1,7 @@
+schema_version: 1
+name: jboss.container.wildfly.launch.access-log-valve
+version: '1.0'
+description: Configures the access-log valve and log handler
+execute:
+  - script: configure.sh
+    user: '185'
```

Comment:
It would be great if we add the used environment variables now we have them refreshed, later just linking this file in wildfly-s2i will supply the WildFly documentation.

We can add them later, it is just to do it all at once.

Reply:
especially the new ones